Update Onboarding Checklist (#1576)
cloud-gov/product: OnboardingChecklist.md

- [ ] Join [the cloud.gov team Google Group](https://groups.google.com/a/gsa.gov/forum/?hl=en#!forum/cloud-gov) so you can participate in team-wide internal communication.
- [ ] Business Unit Only - Join the [cloud.gov inquiries Google Group](https://groups.google.com/a/gsa.gov/forum/#!forum/cloud-gov-inquiries) so you can keep apprised of prospective new clients.
- [ ] Add the [cloud.gov Google Drive folder](https://drive.google.com/drive/folders/0Bx6EvBXVDWwheUtVckVnOE1pRzA) to your Google Drive -- that's where we put cloud.gov docs. If you create or move a doc there, it'll get the right access permissions for team members to be able to view and edit it.
- [ ] Subscribe to [the cloud.gov team calendar](https://calendar.google.com/calendar/embed?src=gsa.gov_0samf7guodi7o2jhdp0ec99aks@group.calendar.google.com&amp;ctz=America/Los_Angeles) (click the + in the bottom right) so you know when assorted team meetings are happening in the various squads. Tip: When you plan Out of Office time, make a calendar event for that on the cloud.gov calendar so that your teammates know you'll be away

> - [ ] Subscribe (through the GitHub watch function) to the [cg-site](https://github.com/18F/cg-site) GitHub repository notifications

Reviewer: Let's remove this:

```suggestion
```

#### Fulfill security and compliance requirements (including for FedRAMP) - Completed by onboarding buddy

- [ ] Make sure they're in [the list of people working on the project](https://docs.google.com/spreadsheets/d/1mW3tphZ98ExmMxLHPogSpTq8DzYr5Oh8_SHnOTvjRWM/edit#gid=0).
- [ ] Add their name, whether they're Cloud Ops (Platform), and the date they joined the team to the [training tracker](https://docs.google.com/spreadsheets/d/1hqU6cNeEB293OT0j3OvbdAFRkrf2zDOrPVxGfnr4sSw/edit#gid=0). Copy the formulas for the due dates from an existing row (grab the "corner" of the cells and pull down).
- [ ] As they complete training, fill out their completion dates in the [training tracker](https://docs.google.com/spreadsheets/d/1hqU6cNeEB293OT0j3OvbdAFRkrf2zDOrPVxGfnr4sSw/edit#gid=0).
- [ ] Add them to the @cloud-gov-team [in Slack’s Team Directory](https://get.slack.help/hc/en-us/articles/212906697-User-Groups#edit-a-user-group).

> - [ ] Add them to the recurring cloud.gov meetings that are relevant for them in [the team calendar](https://calendar.google.com/calendar/embed?src=gsa.gov_0samf7guodi7o2jhdp0ec99aks@group.calendar.google.com&amp;ctz=America/Los_Angeles).

Reviewer: This is more of a review step once they're added to the cloud.gov Google Groups, but good to check!

```suggestion
- [ ] Review the recurring cloud.gov meetings that are relevant for them in [the team calendar](https://calendar.google.com/calendar/embed?src=gsa.gov_0samf7guodi7o2jhdp0ec99aks@group.calendar.google.com&amp;ctz=America/Los_Angeles) (they will get access to this when added to the cloud.gov Team Google Group).
```

> - [ ] Ask one of our cloud-gov team Github Maintainers to add them to the [@18F/cloud.gov team](https://github.com/orgs/18F/teams/cloud-gov) on GitHub.

Reviewer: Let's update this to be the current org and team in GitHub:

```suggestion
* [ ] Add them to the [`cloud-gov`](https://github.com/orgs/cloud-gov/people) organization in GitHub, and the [`cloud-gov-team`](https://github.com/orgs/cloud-gov/teams/cloud-gov-team) team.
```

#### Learn our policies and procedures

> * [ ] Coordinate with your onboarding buddy to go through [Incident Response Training](https://docs.google.com/presentation/d/1AZjQE8zBzMRWZIFUuJPkJLted1ykGtALrLPoPRx5Vls/edit#slide=id.p) within 60 days of joining the team (and annually after that). This will cover the following document, which you should also review before or after training:
> * [ ] Read the [Incident Response Guide](https://cloud.gov/docs/ops/security-ir/).
> * [ ] Coordinate with your onboarding buddy to go through [nonpublic information training](https://docs.google.com/presentation/d/1uB4MlGCu8ZYUxjKVZKwicQ95MvLxaT4Mh93y6w79GPw/edit#slide=id.p) within 60 days of joining the team (and annually after that). This will cover the following documents, which you should also review before or after training:
>   * [ ] Review the [18F open source policy guidance about protecting sensitive information](https://github.com/18F/open-source-policy/blob/master/practice.md#protecting-sensitive-information).
>   * [ ] Read our [sharing secret keys](https://cloud.gov/docs/ops/secrets/#sharing-secret-keys) policy.
>   * [ ] Review the [18F requirements for password management](https://handbook.tts.gsa.gov/password-requirements/).
> * [ ] Coordinate with your onboarding buddy to go through Contingency Planning training within 10 days (and annually after that). This will cover the following document, which you should also review before or after training:
> * [ ] Read the [Contingency Plan](https://docs.cloud.gov/ops/contingency-plan/).

Reviewer: Let's reorder this slightly:

```suggestion
* [ ] Coordinate with your onboarding buddy to go through Contingency Planning training within 10 days (and annually after that). This will cover the following document, which you should also review before or after training:
* [ ] Read the [Contingency Plan](https://docs.cloud.gov/ops/contingency-plan/).
* [ ] Coordinate with your onboarding buddy to go through [Incident Response Training](https://docs.google.com/presentation/d/1AZjQE8zBzMRWZIFUuJPkJLted1ykGtALrLPoPRx5Vls/edit#slide=id.p) within 60 days of joining the team (and annually after that). This will cover the following document, which you should also review before or after training:
* [ ] Read the [Incident Response Guide](https://cloud.gov/docs/ops/security-ir/).
* [ ] Coordinate with your onboarding buddy to go through [nonpublic information training](https://docs.google.com/presentation/d/1uB4MlGCu8ZYUxjKVZKwicQ95MvLxaT4Mh93y6w79GPw/edit#slide=id.p) within 60 days of joining the team (and annually after that). This will cover the following documents, which you should also review before or after training:
  * [ ] Review the [18F open source policy guidance about protecting sensitive information](https://github.com/18F/open-source-policy/blob/master/practice.md#protecting-sensitive-information).
  * [ ] Read our [sharing secret keys](https://cloud.gov/docs/ops/secrets/#sharing-secret-keys) policy.
  * [ ] Review the [18F requirements for password management](https://handbook.tts.gsa.gov/password-requirements/).
```

Your onboarding buddy should add you to the following private channels:

- [ ] `#cg-incidents` - private channel for incident response
- [ ] `#cg-supportstream` - notification channel from our support system, ZenDesk
- [ ] `#cg-priv-all` - private channel for in-team discussion
- [ ] `#cg-priv-gov` (Federal employees only) - may contain discussion of contracting-related or other private, federal-employee-only comms

## Platform-Ops-specific items

### Account access

*Note: These are all contingent on completing the GSA Mandatory Cyber Security and Privacy Training first. AWS user names should be identical across accounts so that permissions can be correctly managed by Terraform.*

* [ ] [AWS Accounts](https://cloud.gov/docs/ops/aws-onboarding/) via the AWS web console (not Terraform) and provide one-time credentials.
* [ ] AWS GovCloud
* [ ] AWS East tied to GovCloud
* [ ] Add their account name to the [audit input file](https://github.com/cloud-gov/cg-compliance/blob/master/audit/inputs.yml)
* [ ] Nessus Manager GUI
* [ ] [Make them an admin](https://cloud.gov/docs/ops/managing-users/#managing-admins) of the platform.

> * [ ] Add them to the *cloud-gov-operators* organization in cloud.gov.

Reviewer: Let's update this link to the current team:

```suggestion
* [ ] Add them to the [`platform-ops`](https://github.com/orgs/cloud-gov/teams/platform-ops) team in GitHub.
```

  - `brew install cloudfoundry/tap/bosh-cli`
  - [ ] Verify the installation using
    - `bosh -v`
- [ ] Install Terraform and other tools per [cg-provision](https://github.com/18F/cg-provision)
  - `brew install terraform`
  - `brew install awscli`
  - `brew install jq`
  - [ ] Verify Terraform installed and is in your path
    - Run `terraform` and helper text should display
  - [ ] Verify AWS CLI installed and is in your path
    - Run `aws` and helper text should display
- [ ] Install and configure `aws-vault` by [following our directions](https://cloud.gov/docs/ops/secrets/#install-aws-vault-for-aws-credentials-and-create-a-profile)
- [ ] Install the Concourse `fly` CLI
  - Download the `fly` binary zip for MacOS from https://concourse-ci.org/
  - Extract the binary and move it to `/usr/local/bin/fly` so it's in your path
    - `cd ~/Downloads`
    - `mv fly /usr/local/bin/fly`
  - [ ] Verify using `fly -h`
- [ ] Install cloud.gov dev tools by cloning the [`cg-scripts` repo](https://github.com/18F/cg-scripts/)
  - `git clone https://github.com/18F/cg-scripts.git`

> ### If developing or working on cloud.gov site content

Reviewer: This section no longer needed:

```suggestion
```

> - [ ] Review the [dashboard contributing guide](https://github.com/18F/cg-dashboard/blob/master/CONTRIBUTING.md) and [`cg-style` standards](https://github.com/18F/cg-style/blob/master/documentation/frontend_standards.md) which help frame what we're looking for in code review
> - [ ] Set up the [landing/docs page](https://github.com/18F/cg-site), [dashboard](https://github.com/18F/cg-dashboard), and [`cg-style`](https://github.com/18F/cg-style) locally
> - [ ] Set up `cg-style` to be [linked to the other sites locally](https://github.com/18F/cg-style#development-and-contributing-setup)
> - [ ] For additional development environment setup, see [setting up a local development environment](https://cloud.gov/docs/ops/setting-up-a-local-development-environment/), which contains setup instructions for GSA Macs

Reviewer: Removing section:

```suggestion
```
